_includes/page-footer.html

```diff
@@ -14,18 +14,8 @@ <h3>Get in touch for a demo or to learn more.</h3>
 </div>
 
 <div class="container-fluid bg-darkblue">
-  <div class="container p-0 carousel slide" id="carousel" data-ride="carousel" data-interval="9000">
+  <div class="container p-0 carousel carousel-dark slide" id="carousel" data-bs-ride="carousel" data-bs-interval="9000">
```

Comment:
How does the `carousel-dark` style affect the carousel?

Reply:
It appears to mainly make the arrows light colored instead of dark.